frontend/src/scenes/dashboard/dashboardLogic.js

```diff
@@ -67,7 +70,7 @@ export const dashboardLogic = kea({
                     })
                 })
 
-                return state.map(item => ({ ...item, layouts: itemLayouts[item.id] }))
+                return { ...state, items: state.items.map(item => ({ ...item, layouts: itemLayouts[item.id] })) }
```

Comment:
This breaks changing anything else on the dashboard (e.g. error when trying to change a color), since the lines just below all expect `state` to be an array... and they don't know about `state.items`.

(This is something that should have been caught by a good test for the dashboards...)